File: commons/src/main/java/com/eirsteir/coffeewithme/commons/domain/friendship/FriendRequestAcceptedEvent.java
package com.eirsteir.coffeewithme.commons.domain.friendship;

import com.eirsteir.coffeewithme.commons.domain.notification.AbstractEntityNotificationEvent;
import com.eirsteir.coffeewithme.commons.domain.user.UserDetails;
import lombok.NoArgsConstructor;

@NoArgsConstructor
public class FriendRequestAcceptedEvent extends AbstractEntityNotificationEvent implements FriendshipEvent {

    public FriendRequestAcceptedEvent(Long subjectId, UserDetails user) {
        super(subjectId, user);
    }

}

File: commons/src/main/kotlin/com/eirsteir/coffeewithme/commons/domain/friendship/FriendRequestAcceptedEvent.kt
package com.eirsteir.coffeewithme.commons.domain.friendship

import com.eirsteir.coffeewithme.commons.domain.notification.AbstractEntityNotificationEvent
import com.eirsteir.coffeewithme.commons.domain.user.UserDetails

class FriendRequestAcceptedEvent(subjectId: Long, user: UserDetails) :
    AbstractEntityNotificationEvent(subjectId, user), FriendshipEvent